src/views/HowToPlay/Layout/StyledInstructionsBody.tsx

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
```

Comment:
I would delete this whole file, I don't think you need this extra component.

Reply:
I needed to keep this file because it's the only way I can apply margin to the text section and the back button, but not the Title Banner. I moved all the main text margin there.

Reply:
🖕

Reply:
Ah I see. Later we could create a root layout with the TitleBar and an Outlet component. Then you could get rid of this one.